content/rest-api/filter.md

```diff
@@ -369,6 +384,10 @@ You can even combine several filters on the same attribute. The example below wi
 /api/rest/v1/products?search={"description":[{"operator":"NOT EMPTY","locale":"en_US","scope":"ecommerce"},{"operator":"EMPTY","locale":"fr_FR","scope":"ecommerce"}]}
 ```
 
+:::info
+Filtering on product values is also available for published products.
```

Comment:
As it has no meaning to use this filter, let's remove this mention.

Reply:
I'm sorry I don't understand why :/ In this case, the two mentions above has also no sense, no?